src/Widgets/AlbumListGrid.vala

```diff
@@ -39,18 +39,14 @@ public class Noise.AlbumListGrid : Gtk.Grid {
         }
     }
 
-    Widgets.AlbumImage album_cover;
-    Gtk.Label album_label;
-    Gtk.Label artist_label;
-
-    Gtk.Menu cover_action_menu;
-    Gtk.MenuItem cover_set_new;
-
-    Granite.Widgets.Rating rating;
```

Comment:
Does removing this belongs there or to #153?

Reply:
It's just moved down here as `private Granite.Widgets.Rating rating;` on line 49 :)

Reply:
Oh, sorry, I didn't saw that. :grin: